dts/common/nordic/nrf7120_enga.dtsi

```diff
@@ -0,0 +1,846 @@
+				reg = <0xee000 0x1000>;
+				interrupts = <238 NRF_DEFAULT_IRQ_PRIORITY>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				clock-frequency = <I2C_BITRATE_STANDARD>;
+				easydma-maxcnt-bits = <16>;
+				status = "disabled";
+			};
+
+			uart24: uart@ee000 {
+				compatible = "nordic,nrf-uarte";
+				reg = <0xee000 0x1000>;
+				interrupts = <238 NRF_DEFAULT_IRQ_PRIORITY>;
+				status = "disabled";
+				endtx-stoptx-supported;
+				frame-timeout-supported;
+			};
+
+			dppic30: dppic@102000 {
+				compatible = "nordic,nrf-dppic";
+				reg = <0x102000 0x1000>;
+				status = "disabled";
+			};
+
+			ppib30: ppib@103000 {
+				compatible = "nordic,nrf-ppib";
+				reg = <0x103000 0x1000>;
+				status = "disabled";
+			};
+
+			spi30: spi@104000 {
+				compatible = "nordic,nrf-spim";
+				reg = <0x104000 0x1000>;
+				interrupts = <260 NRF_DEFAULT_IRQ_PRIORITY>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				max-frequency = <DT_FREQ_M(8)>;
+				easydma-maxcnt-bits = <16>;
+				rx-delay-supported;
+				rx-delay = <1>;
+				status = "disabled";
+			};
+
+			i2c30: i2c@104000 {
+				compatible = "nordic,nrf-twim";
+				reg = <0x104000 0x1000>;
+				interrupts = <260 NRF_DEFAULT_IRQ_PRIORITY>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				clock-frequency = <I2C_BITRATE_STANDARD>;
+				easydma-maxcnt-bits = <16>;
+				status = "disabled";
+			};
+
+			uart30: uart@104000 {
+				compatible = "nordic,nrf-uarte";
+				reg = <0x104000 0x1000>;
+				interrupts = <260 NRF_DEFAULT_IRQ_PRIORITY>;
+				status = "disabled";
+				endtx-stoptx-supported;
+				frame-timeout-supported;
+			};
+
+			comp: comp@106000 {
+				compatible = "nordic,nrf-comp";
+				reg = <0x106000 0x1000>;
+				interrupts = <262 NRF_DEFAULT_IRQ_PRIORITY>;
+				#io-channels-cells = <1>;
+				status = "disabled";
+			};
+
+			wdt31: wdt@109000 {
+				compatible = "nordic,nrf-wdt";
+				reg = <0x109000 0x1000>;
+				interrupts = <265 NRF_DEFAULT_IRQ_PRIORITY>;
+				status = "disabled";
+			};
+
+			gpio0: gpio@10a000 {
+				compatible = "nordic,nrf-gpio";
+				reg = <0x10a000 0x200>;
+				gpio-controller;
+				#gpio-cells = <2>;
+				port = <0>;
+				ngpios = <10>;
+				gpiote-instance = <&gpiote30>;
+				status = "disabled";
+			};
+
+			gpiote30: gpiote@10c000 {
+				compatible = "nordic,nrf-gpiote";
+				reg = <0x10c000 0x1000>;
+				instance = <30>;
+				status = "disabled";
+			};
+
+			clock: clock@10e000 {
+				compatible = "nordic,nrf-clock";
+				reg = <0x10e000 0x1000>;
+				interrupts = <270 NRF_DEFAULT_IRQ_PRIORITY>;
+				status = "disabled";
+			};
+
+			cpuapp_ppb: cpuapp-ppb-bus {
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				cpuapp_nvic: interrupt-controller@e000e100  {
+					#address-cells = <1>;
+					compatible = "arm,v8m-nvic";
+					reg = <0xe000e100 0xc00>;
+					arm,num-irq-priority-bits = <3>;
+					interrupt-controller;
+					#interrupt-cells = <2>;
+				};
+
+				cpuapp_systick: timer@e000e010 {
```

Comment:
This node should be placed before https://github.com/nrfconnect/sdk-nrf/blob/780e06b375fa2ac576c0d5a500eb9ca4c97deea4/dts/common/nordic/nrf7120_enga.dtsi#L817